features/minion/repository/src/main/resources/features.boot

```diff
@@ -11,6 +11,7 @@ opennms-trapd-listener
 opennms-events-sink-dispatcher
 opennms-send-event-command
 opennms-dnsresolver-netty
+opennms-deviceconfig
```

Comment:
Should the `opennms-deviceconfig` feature really be enabled by default on Minion?

We also have to discuss the different features we need. I think there should not be a general feature `opennms-deviceconfig`. We have poller/sink and server/client related bundles. I did some "featuring" in branch `jira/NMS-13937`.

Reply:
yeah, I realized all the monitors need to be enabled by default on Minion.

Reply:
WDYT on the proposed renames, then?

Reply:
`opennms-deviceconfig-monitor` for the Monitor and `opennms-deviceconfig-sink` for the Sink